### ISSUE TITLE
Paladin Prot Spells Not Working Properly

### ISSUE THREAD

**Issue by Sendoakagii** [dobidoberman1/Mystic-5.4.8-Bug-Tracker#18]
Paladin Prot Spells Not Working Properly 3 that doesn't . Only Boss Like encounters.(This like Arena an daily like boss)
1. [Hammer of the Righteous] 
2..[Hammer of Wrath]
3.. [Shield of the Righteous]

What happens is that when in combat with boss it may hit once but then after that it just say invalid target . or target is not in front of you

**Comment by dobidoberman1** [comment]
Can you rephrase that please Sendo man? I got confused.

**Comment by Sendoakagii** [comment]
Ok when casting those spell on arenas like ring of blood an  The Amphitheater of Anguish. the game says the target is not in front of you . an so after that you pretty much useless on those questing areas

**Comment by dobidoberman1** [comment]
Why though? who does it says that? I mean are you actually standing infront of a mob while it pop this issue? like you're facing him but the issue comes up?

**Comment by Sendoakagii** [comment]
YES thats my problem i stand directly infront of the mob an it pops your target is not infront of you an this also happen with blood strike on the death knghts

**Comment by dobidoberman1** [comment]
Alright I think i fixed it, we'll test it after next restart.
